composer.json

```diff
@@ -17,18 +17,18 @@
         }
     ],
     "require": {
-        "php": "^5.3 || ^7.0",
+        "php": "^7.1",
```

Comment:
"^2.8" didn't require 7.1, why to change version of php?
Even symfony 3.4 supports php 7.0....

Reply:
I think we reverted that, didn't we?

Reply:
I didn't find revert commit.
Thing is that I can use 4.x version, because of php7.0 (I have restrictions for php upgrade)
Version 3.x I can use because of symfony 2.8 (I have symfony 3.4)

And as I see in 4.x : void types are already in use. :)

Reply:
Found "Restore php 5.6 through 7.0 compatibility", but symfony version is 2.8 there, so it doesn't suits me too.

Reply:
Yes, that happened to me aswell, you need to either use SF 2.8 and php 5.6  or upgrade php to 7.1 to use SF 3.4

Reply:
Uuuuh right... too bad. If I were you I'd upgrade to 7.1, 7.0 is not supported anymore, apart for security fixes: http://php.net/supported-versions.php

Reply:
Ok. I see.
Then I will try to continue with custom solution:
https://github.com/chernecov/SonataUserBundle/tree/symfony3-php70-compatibility
Will wait for company to choose higher version of php. 
Thanks a lot for quick answers.